barrantes/book_journal.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Book Details</title>
+</head>
+<body>
+    <h1>Book Details</h1>
+    <h2>Create Book Entry</h2>
+    <form id="createForm">
```

Comment:
IDs should be written in snake case. All lower case separated by underscore. For reference, please click on the link provided below to access further information.

```suggestion
    <form id="create_form">
```


[https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx- TPQxdLySow/edit#slide=id.g24da8de92d7_0_5](https://github.com/PUP-BSIT/ex16-tech_nx/pull/1/commits/url)